app/assets/javascripts/admin/addon/components/admin-branding-fonts-form.gjs

```diff
@@ -0,0 +1,267 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { fn } from "@ember/helper";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import { classify, decamelize, underscore } from "@ember/string";
+import DButton from "discourse/components/d-button";
+import Form from "discourse/components/form";
+import UpdateDefaultTextSize from "discourse/components/modal/update-default-text-size";
+import concatClass from "discourse/helpers/concat-class";
+import { ajax } from "discourse/lib/ajax";
+import { DEFAULT_TEXT_SIZES, FONTS } from "discourse/lib/constants";
+import { bind } from "discourse/lib/decorators";
+import { i18n } from "discourse-i18n";
+import eq from "truth-helpers/helpers/eq";
+
+const MAIN_FONTS = [
+  "Helvetica",
+  "Inter",
+  "Lato",
+  "Montserrat",
+  "OpenSans",
+  "Poppins",
+  "Roboto",
+  "Merriweather",
+  "Mukta",
+];
+const MORE_FONTS = FONTS.filter((font) => !MAIN_FONTS.includes(font));
```

Comment:
I think we should just store this logic on the server side. This is pretty much a constant and doesn't need to be computed at runtime.